pkg/operator/ceph/cluster/controller.go

```diff
@@ -254,6 +256,15 @@ func (c *ClusterController) onAdd(obj interface{}) {
 		logger.Warningf("mon count is even (given: %d), should be uneven, continuing", cluster.Spec.Mon.Count)
 	}
 
+	// When the dataDirHostPath is set, check if the path is in the disallowed paths list
+	if cluster.Spec.DataDirHostPath != "" {
```

Comment:
Shall we call `path.Clean(cluster.Spec.DataDirHostPath)` before comparing? Otherwise there could be simple differences like a trailing slash.

Reply:
Done. I had shortly thought about using `strings.HasPrefix()` but that would break anyone using those paths in anyway in the right now.